src/lib/gundb/FeedStorage.js

```diff
@@ -120,12 +120,10 @@ export class FeedStorage {
     this.userStorage = userStorage
     this.walletAddress = wallet.account.toLowerCase()
     log.debug('initialized', { wallet: this.walletAddress })
-    this.ready = new Promise((resolve, reject) => {
```

Comment:
why this has been reverted ? it looks logically
maybe here was just a small mistake required a small fix